lib/pdk/validators/base_validator.rb

```diff
@@ -47,8 +59,29 @@ def self.spinner_text(_targets = nil)
       end
 
       def self.invoke(report, options = {})
-        targets = parse_targets(options)
+        targets, skipped, invalid = parse_targets(options)
 
+        skipped.each do |skipped_target|
+          PDK.logger.debug(_("%{validator} skipped unmatched target: '%{target}'") % { validator: name, target: skipped_target })
+          report.add_event(
+            file:     skipped_target,
+            source:   name,
+            message:  'Target skipped. No files to validate.',
```

Comment:
From the code above; I believe that _skipped_ means that the file/directory given by the user as a target does exist on the filesystem, but does not match the pattern built into the validator.
I'm not sure this message conveys that. And 'No files to validate' might be confusing if other targets in the command did have valid targets.

    message: 'Target was skipped as it does not match the validator\'s pattern'

?